java/src/com/ibm/streamsx/rest/primitives/JobGson.java

```diff
@@ -0,0 +1,39 @@
+/*
+# Licensed Materials - Property of IBM
+# Copyright IBM Corp. 2017
+ */
+package com.ibm.streamsx.rest.primitives;
+
+import java.util.ArrayList;
+
+class JobGson {
```

Comment:
Not sure such a class is needed. I thought Gson could automatically create instances of a class from JSON.